Update model specs and pricing for DeepSeek models

samurai00 opened the pull request:
#### 💻 变更类型 | Change Type

<!-- For change type, change [ ] to [x]. -->

- [ ] ✨ feat
- [x] 🐛 fix
- [ ] ♻️ refactor
- [ ] 💄 style
- [ ] 👷 build
- [ ] ⚡️ perf
- [ ] 📝 docs
- [ ] 🔨 chore

#### 🔀 变更说明 | Description of Change

- 根据 OpenRouter 官网调整其 DeepSeek V3/R1 模型的配置与计费价格
- 恢复火山引擎中 DeepSeek R1 模型的定价，以便正确显示积分预览

<!-- Thank you for your Pull Request. Please provide a description above. -->

#### 📝 补充信息 | Additional Information

<!-- Add any other context about the Pull Request here. -->

samurai00 committed:
Update model specs and pricing for DeepSeek models

  - Adjust configurations and token pricing for DeepSeek V3/R1 in OpenRouter
  - Restore original pricing for DeepSeek R1 models in VolcEngine

canisminor1990 commented:
👍 @samurai00

Thank you for raising your pull request and contributing to our Community
Please make sure you have followed our contributing guidelines. We will review it as soon as possible.
If you encounter any problems, please feel free to connect with us.\
非常感谢您提出拉取请求并为我们的社区做出贡献，请确保您已经遵循了我们的贡献指南，我们会尽快审查它。
如果您遇到任何问题，请随时与我们联系。

samurai00 committed:
Update model descriptions and pricing for DeepSeek V3 and R1

- Revised descriptions for DeepSeek V3 in both English and Chinese locales to reflect its advanced capabilities and performance.
- Adjusted pricing for DeepSeek V3 and R1 models in OpenRouter configuration.
- Updated context window tokens for DeepSeek V3 in model provider settings.

arvinxx committed:
Merge branch 'main' into fix-deepseek-models-price

samurai00 commented:
🌋火山价格已经在 main 分支修正了。
